cypress/fixtures/create_post_success.json

```diff
@@ -0,0 +1,4 @@
+{
+  "message": "Successfully created",
+  "post.id": 1
```

Comment:
I think that this should look like this: 

```
{
"message": "Successfully created",
  "post": {"id": 1}
}

Reply:
Yes, seems like we thought about the format from Ruby perspective..